### PR TITLE
upgrade iceberg to org.apache.iceberg:iceberg-spark-runtime

### PR DESCRIPTION
**Pull request by zhljen**

### CODE REVIEW

File: metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/iceberg/IcebergTableHandler.java [hunk 1]
```diff
@@ -207,11 +218,11 @@ public boolean update(final TableInfo tableInfo) {
             }
             if (result) {
                 updateSchema.commit();
-                final String newTableMetadataLocation = icebergMetastoreTables.getTableOps().currentMetadataLocation();
```
Comment (ajoymajumdar) [inline metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/iceberg/IcebergTableHandler.java]: Use the method currentMetadataLocation. The method needs to be overwritten in our extended class below.

File: metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/iceberg/IcebergTableHandler.java [hunk 2]
```diff
@@ -322,8 +361,8 @@ public TableMetadata refresh() {
         }
 
         @Override
-        public String currentMetadataLocation() {
```
Comment (ajoymajumdar) [inline metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/iceberg/IcebergTableHandler.java]: Why did we remove this?

File: metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/iceberg/IcebergTableHandler.java (hunk 1 above)
Comment (zhljen) [reply metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/iceberg/IcebergTableHandler.java]: that's the changing I am working on. will update

File: metacat-metadata-mysql/src/test/groovy/com/netflix/metacat/metadata/mysql/MysqlUserMetadataServiceSpec.groovy [hunk 3]
```diff
@@ -24,6 +25,7 @@ import java.time.Instant
  * TODO: Need to move this to integration-test
  * @author amajumdar
  */
+@Ignore
```
Comment (ajoymajumdar) [inline metacat-metadata-mysql/src/test/groovy/com/netflix/metacat/metadata/mysql/MysqlUserMetadataServiceSpec.groovy]: Can we add this back if it is running successfully in travis.

File: metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/iceberg/IcebergTableHandler.java (hunk 1 above)
Comment (zhljen) [reply metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/iceberg/IcebergTableHandler.java]: Added back this method.

**Pull request merged**